custom/icds_reports/migrations/sql_templates/database_views/ccs_record_monthly_view.sql

```diff
@@ -73,7 +73,11 @@ CREATE VIEW ccs_record_monthly_view AS
         "ccs_record_monthly"."date_death" AS "date_death",
         "ccs_record_monthly"."eating_extra" AS "eating_extra",
         "ccs_record_monthly"."resting" AS "resting",
-        "ccs_record_monthly"."immediate_breastfeeding" AS "immediate_breastfeeding"
+        "ccs_record_monthly"."immediate_breastfeeding" AS "immediate_breastfeeding",
+        "ccs_record_monthly"."caste" AS "caste",
+        "ccs_record_monthly"."disabled" AS "disabled",
+        "ccs_record_monthly"."minority" AS "minority",
+        "ccs_record_monthly"."resident" AS "resident"
```

Comment:
can you add these fields to this model as well https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/models/views.py#L633

Reply:
I already have all those fields from `CcsRecordMonthly`, isn't that what creates the view?

Reply:
the model that I linked is representing to ccs_record_monthly_view called CcsRecordMonthlyView https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/models/views.py#L711

So its the django representation of the view which is used in some reports/exports. they won't be used for now, but its good to keep it up to date

Reply:
Thanks for the clarification: https://github.com/dimagi/commcare-hq/pull/23552/commits/883c71409955fdc7b42d599f213237c8bb7670f5